gitguard.py

```diff
@@ -301,18 +301,21 @@ def get_commit_history_for_file(repo_link, file_path, author_name=None, start_li
         list of commits in format [sha, title] if author's name is given
     """
     repo = _clone_repo(repo_link)
-    if start_line and end_line: 
-        lines_limit = "%s,%s" % (start_line, end_line)
-        if not author_name:
-            log = repo.git.log(L= lines_limit, pretty = "%H\t%an\t%s", file= file_path)
-        else:
-            log = repo.git.log(L= lines_limit, pretty = "%H\t%an\t%s", author = author_name, file= file_path)
+
+    lines_limit = ""
+    if not start_line:
+        start_line = 1
+    if not end_line:
+        lines_limit = "%s" % (file_path)
```

Comment:
my is lines_limit a string? I was expecting an integer
you can just use file_path if it's already a string, or str(file_path)